### PR TITLE
New-package: ironbar-0.14.1

### PR DESCRIPTION
**Pull request by caughtquick**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by caughtquick** [comment]
Should I be setting the specific arches where this package builds or is it fine?

File: srcpkgs/ironbar/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'ironbar'
+pkgname=ironbar
+version=0.14.0
+revision=1
+build_style=cargo
+makedepends="gtk+3-devel gtk-layer-shell-devel pkg-config"
```
Comment (oreo639) [inline srcpkgs/ironbar/template:6]: pkg-config should be in hostmakedpends.

**Comment by caughtquick** [comment]
I've corrected that issue, and it appears like it's building on all systems now, thanks for the help @oreo639.

File: srcpkgs/ironbar/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+maintainer="caughtquick <abhijit@sipahimalani.me>"
+license="MIT"
+homepage="https://github.com/JakeStanger/ironbar"
+changelog="https://github.com/JakeStanger/ironbar/releases/"
+distfiles="https://github.com/JakeStanger/ironbar/archive/refs/tags/v${version}.tar.gz"
+checksum=5cfea20846c41ca294e7093e045f7e0efa7036ef9df2f0003717c5ca61587c1d
+
+post_install() {
+	vlicense LICENSE
+	}
```
Comment (ahesford) [inline srcpkgs/ironbar/template:18]: Remove indentation of the closing brace.

**Review by ahesford — CHANGES_REQUESTED** [review]
Looks like 0.14.1 is out now.

**Comment by ahesford** [comment]
Also, I think you should install the contents of the `examples/` directory. The `vsconf` helper is convenient for this.

**Pull request merged**